workspaces/cli-client/src/client.ts

```diff
@@ -23,10 +23,16 @@ class Client {
   findSession(
```

Comment:
I actually asked @devdoshi about this.. turns out that the actual functionality here is to create a session if it doesn't exist, otherwise look it up, and return it. So it... _is_ a mutating function... but I also wanted to rename it when I saw it too, so I'm not totally sure what to do about it.

Reply:
getOrCreateSession? I think we have getOrCreateAnonId in other parts of the code